Received ExecOnConflictUpdate error after running `make process`

**Description**
This error is happening starting today, I was unsure if the cause might be my branch so I backup'd my branch work and moved to `main`. The error persists, so you can assume this error is happening in the `main` branch.

I got the error mentioned in this description after running the command `make process`, making sure the database is properly running and the migration has been previously executed.

Something important that I would like to clarify is that I'm using postgres in docker, not sure if you have postgres binaries already working in your local machine instead of a docker container.

**Prerequisites**

I made sure to execute exactly the same steps mentioned [in the documentation](https://github.com/ChainSafe/Multix/blob/main/squid/README.md#quickly-running-the-sample) in order to run the app.

But for reference, this is the list of commands I ran, properly ordered by execution.

```sh

make ci
make codegen
make typegen
make build
make up
make migrate
make process

# after a minute, the command exits with an error
```

**Steps To Reproduce**

I would suggest to run the application again from scratch, using the steps documented [here](https://github.com/ChainSafe/Multix/blob/main/squid/README.md#quickly-running-the-sample), is the reference I'm using to start the app and eventually, deploy the application.

**Expected Behaviour**
-

**Console Error**
If possible, please copy/paste or provide a screenshot of the browser console. To do so, press `Ctrl` + `Shift` + `J` (or `Cmd` + `Shift` + `J` on a Mac)  

```
15:37:25 FATAL sqd:processor QueryFailedError: ON CONFLICT DO UPDATE command cannot affect row a second time
                                 at PostgresQueryRunner.query (/Users/ronny/Code/chainsafe/multix/squid/node_modules/typeorm/driver/postgres/PostgresQueryRunner.js:211:19)
                                 at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
                                 at async InsertQueryBuilder.execute (/Users/ronny/Code/chainsafe/multix/squid/node_modules/typeorm/query-builder/InsertQueryBuilder.js:106:33)
                                 at async Store.upsertMany (/Users/ronny/Code/chainsafe/multix/squid/node_modules/@subsquid/typeorm-store/lib/store.js:70:13)
                                 at async Store.saveMany (/Users/ronny/Code/chainsafe/multix/squid/node_modules/@subsquid/typeorm-store/lib/store.js:57:13)
                                 at async Store.upsert (/Users/ronny/Code/chainsafe/multix/squid/node_modules/@subsquid/typeorm-store/lib/store.js:31:13)
                                 at async handleNewMultisigCalls (/Users/ronny/Code/chainsafe/multix/squid/lib/processorHandlers/handleNewMultisigCalls.js:21:5)
                                 at async /Users/ronny/Code/chainsafe/multix/squid/lib/processor.js:111:32
                                 at async TypeormDatabase.runTransaction (/Users/ronny/Code/chainsafe/multix/squid/node_modules/@subsquid/typeorm-store/lib/database.js:110:13)
                                 at async TypeormDatabase.transact (/Users/ronny/Code/chainsafe/multix/squid/node_modules/@subsquid/typeorm-store/lib/database.js:64:24)
                                 query: INSERT INTO "multisig_call"("id", "block_hash", "timestamp", "call_index", "multisig_id") VALUES ($1, $2, $3, $4, $5), ($6, $7, $8, $9, $10), ($11, $12, $13, $14, $15), ($16, $17, $18, $19, $20), ($21, $22, $23, $24, $25), ($26, $27, $28, $29, $30), ($31, $32, $33, $34, $35), ($36, $37, $38, $39, $40), ($41, $42, $43, $44, $45) ON CONFLICT ( "id" ) DO UPDATE SET "id" = EXCLUDED."id", "block_hash" = EXCLUDED."block_hash", "timestamp" = EXCLUDED."timestamp", "call_index" = EXCLUDED."call_index", "multisig_id" = EXCLUDED."multisig_id"
                                 parameters: ["5G1UwbpmvtMvadJtDacokB8yA235yiXr94PBjoaqUdBtnGEG-5037705-2","0x96b715e1b96972b26c837ac9e63dc7010f17c88190c521c015193d9466be4881","2023-04-20T09:03:24.000Z",2,"5G1UwbpmvtMvadJtDacokB8yA235yiXr94PBjoaqUdBtnGEG","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5041713-2","0xe7869cea6c385802ab902bd6d3ddbdf268cd9fc68e7bc502ed973fc87f7f0766","2023-04-20T15:44:12.001Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5042224-2","0x75fc816c6cb7d31a2c2c4448183d48a7baf9b0d8f8ab72b5a82140074f4402a6","2023-04-20T16:35:18.000Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5042262-2","0x57f1efd00ba87ca39267f7bac80db1a75387dc225e5cccce2c4285be9a08bd17","2023-04-20T16:39:06.001Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5042371-2","0x4d109ead350bf5cd266cd79090d3fbefd93eb5dee193e58e1c52f8e2b4e391c7","2023-04-20T16:50:00.001Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5Dut2HsvrRGaKTqGx23DWaRk9quXNJuNiheAfKC3co6rL4C2-5043589-2","0xa42d1974c7423d3c67ac5cdcaa1ac119ad103381d84f910e4e90f6f7c6da2ddf","2023-04-20T18:51:48.000Z",2,"5Dut2HsvrRGaKTqGx23DWaRk9quXNJuNiheAfKC3co6rL4C2","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5044687-2","0x4dcfa70f19434f4aea8cc07b67b4b3ee81de1bd63451a9b0a2bd87540c73ff64","2023-04-20T20:41:36.000Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5044687-2","0x4dcfa70f19434f4aea8cc07b67b4b3ee81de1bd63451a9b0a2bd87540c73ff64","2023-04-20T20:41:36.000Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd","5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd-5044764-2","0xf0fea960cdcba89677135d85ac4ab25f1f7a0cdf92d4f628cb5e9670296ec9f0","2023-04-20T20:49:18.001Z",2,"5GSzS6UNYQrUmuVVp8H8RC29YwWGp7ZzdFbY37E5hcnkwEEd"]
                                 driverError: {"length":239,"name":"error","severity":"ERROR","code":"21000","hint":"Ensure that no rows proposed for insertion within the same command have duplicate constrained values.","file":"nodeModifyTable.c","line":"2552","routine":"ExecOnConflictUpdate","stack":"error: ON CONFLICT DO UPDATE command cannot affect row a second time\n    at Parser.parseErrorMessage (/Users/ronny/Code/chainsafe/multix/squid/node_modules/pg-protocol/dist/parser.js:287:98)\n    at Parser.handlePacket (/Users/ronny/Code/chainsafe/multix/squid/node_modules/pg-protocol/dist/parser.js:126:29)\n    at Parser.parse (/Users/ronny/Code/chainsafe/multix/squid/node_modules/pg-protocol/dist/parser.js:39:38)\n    at Socket.<anonymous> (/Users/ronny/Code/chainsafe/multix/squid/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (node:events:511:28)\n    at addChunk (node:internal/streams/readable:332:12)\n    at readableAddChunk (node:internal/streams/readable:305:9)\n    at Readable.push (node:internal/streams/readable:242:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)"}
                                 length: 239
                                 severity: ERROR
                                 code: 21000
                                 hint: Ensure that no rows proposed for insertion within the same command have duplicate constrained values.
                                 file: nodeModifyTable.c
                                 line: 2552
                                 routine: ExecOnConflictUpdate
make: *** [process] Error 1
```

**Screenshots / Video**
![image](https://user-images.githubusercontent.com/3170758/233493352-bb341aed-bfa6-4b97-993a-2996c4a1e8d3.png)

**Test Host Specifics: **
 - OS: [e.g. macOS 13.3.1]
 - Browser -
 - Version -
 - Wallet -
 
 Node.js version: v20.0.0

**Additional context**
-

--- a/squid/src/processor.ts
+++ b/squid/src/processor.ts
@@ -86,7 +86,7 @@ processor.run(new TypeormDatabase(), async (ctx) => {
         const blockHash = block.header.hash
 
         newMultisigCalls.push({
-          id: getMultisigCallId(newMulti.id, blockNumber, callItem.extrinsic.indexInBlock),
+          id: getMultisigCallId(newMulti.id, blockNumber, callItem.extrinsic.indexInBlock, callItem.call.pos),
           blockHash,
           callIndex: callItem.extrinsic.indexInBlock,
           multisigAddress: newMulti.id,

--- a/squid/src/util/getMultisigCallId.ts
+++ b/squid/src/util/getMultisigCallId.ts
@@ -1,3 +1,3 @@
-export const getMultisigCallId = (address: string, blockNumber: number, extrinsicIndex: number) => {
-  return `${address}-${blockNumber}-${extrinsicIndex}`
+export const getMultisigCallId = (address: string, blockNumber: number, extrinsicIndex: number, position: number) => {
+  return `${address}-${blockNumber}-${extrinsicIndex}-${position}`
 }

